### ISSUE TITLE
Bootstrap release

### ISSUE THREAD

**Issue by bgermann** [okkur/syna#68]
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**Anything else we need to know?**:
Please update bootstrap to the release version.

**Comment by stp-ip** [comment]
Wasn't available at the creation of the theme. Thanks for making us aware.

Sidenote: https://getbootstrap.com/docs/4.0/getting-started/download/

**Comment by bgermann** [comment]
Yes, but that does not include your customizations. How to apply them?

**Comment by stp-ip** [comment]
We cut down the used scss files from bootstrap to create the overwrite: https://github.com/okkur/syna/tree/master/static/css/bootstrap-4.beta/scss-overwrite

They might still work, but we need to create a new set with the v4 as base.

That also enables you to switch the colors.

Sidenote: We will be changing the variable names in v0.11.0 to snake_case. Just to give you a heads-up that a few things might need to be rewritten with v0.11.0. After that we hope that any changes should be mostly backwards compatible.

**Comment by stp-ip** [comment]
Update: https://blog.getbootstrap.com/2018/04/09/bootstrap-4-1/
